benchmarks/DeepLearning/Ops/BatchMatMulRVVOp/GoogleBenchmarkMain.cpp

```diff
@@ -0,0 +1,130 @@
+
+bool areArraysEqual(int array1[], int array2[], int size) {
+  for (int i = 0; i < size; ++i) {
+    if (array1[i] != array2[i]) {
+      return false;
+    }
+  }
+  return true;
+}
+} // namespace
+
+namespace {
+// Declare the batch matmul C interface.
+extern "C" {
+void _mlir_ciface_batch_matmul_scalar(MemRef<int, 3> *input1,
+                                      MemRef<int, 3> *input2,
+                                      MemRef<int, 3> *output);
+void _mlir_ciface_batch_matmul_auto_vectorization(MemRef<int, 3> *input1,
+                                                  MemRef<int, 3> *input2,
+                                                  MemRef<int, 3> *output);
+void _mlir_ciface_batch_matmul_rvv_vectorization(MemRef<int, 3> *input1,
+                                                 MemRef<int, 3> *input2,
+                                                 MemRef<int, 3> *output);
+void _mlir_ciface_batch_matmul_vectorization(MemRef<int, 3> *input1,
+                                             MemRef<int, 3> *input2,
+                                             MemRef<int, 3> *output);
+}
+
+#define DEFINE_BATCH_MATMUL_BENCHMARK(name, func)                              \
+  void BM_BATCH_MATMUL_##name(benchmark::State &state) {                       \
+    intptr_t sizesInput1[3] = {BATCH_SIZE, M, K};                              \
+    intptr_t sizesInput2[3] = {BATCH_SIZE, K, N};                              \
+    intptr_t sizesOutput[3] = {BATCH_SIZE, M, N};                              \
+                                                                               \
+    MemRef<int, 3> input1(sizesInput1, 1.0);                                   \
+    MemRef<int, 3> input2(sizesInput2, 1.0);                                   \
+    MemRef<int, 3> output(sizesOutput, 0.0);                                   \
+                                                                               \
+    for (auto _ : state) {                                                     \
+      func(&input1, &input2, &output);                                         \
+    }                                                                          \
+  }
+
+DEFINE_BATCH_MATMUL_BENCHMARK(SCALAR, _mlir_ciface_batch_matmul_scalar)
+DEFINE_BATCH_MATMUL_BENCHMARK(AutoVectorization,
+                              _mlir_ciface_batch_matmul_auto_vectorization)
+DEFINE_BATCH_MATMUL_BENCHMARK(Vectorization,
+                              _mlir_ciface_batch_matmul_vectorization)
+DEFINE_BATCH_MATMUL_BENCHMARK(RvvVectorization,
+                              _mlir_ciface_batch_matmul_rvv_vectorization)
+} // namespace
+
+// Register benchmark cases.
+BENCHMARK(BM_BATCH_MATMUL_SCALAR)->Unit(benchmark::kMillisecond);
+BENCHMARK(BM_BATCH_MATMUL_AutoVectorization)->Unit(benchmark::kMillisecond);
+BENCHMARK(BM_BATCH_MATMUL_Vectorization)->Unit(benchmark::kMillisecond);
+BENCHMARK(BM_BATCH_MATMUL_RvvVectorization)->Unit(benchmark::kMillisecond);
```

Comment:
`BM_BATCH_MATMUL_RvvVectorization` looks too long, you can consider to remove suffix like `BM` and unify name convention(e.g. BATCH_MATMUL_VECTORIZATION, BATCH_MATMUL_VECTORIZATION_RVV). I will fix other Ops benchmarks' naming conventions soon after.